libs/expo/betterangels/src/lib/services/PlaceAutocomplete/formatParams.ts

```diff
@@ -0,0 +1,47 @@
+import { TLatLngRectangle } from './types';
```

Comment:
Hi @tglaz 

I don't see the change on my end (both iOS and Android). I got the latest build again just in case but I still don't see the fix which means the street number doesn't show up after I tapped a particular address (e.g., 112233 Huntington Drive)

I'd be happy to sync up if you'd like. Thanks.

Reply:
@shiozakiyuko - I commented on this ticket - [DEV-921](https://betterangels.atlassian.net/browse/DEV-921) - about this address. 

Based on what I saw `112233 Huntington Drive` does not seem to exist. Please confirm it, but when I google that address I do not get back and address with a that street number.

Reply:
Hi @tglaz 

I saw your comment in the DEV-921. Thank you. 
Oh ok, Now, I understood the reason and it doesn't seem like the address "112233 Huntington Drive" is existed anywhere in Los Angeles. 

So it's confusing to me that if the address shows up and suggesting with the address, anyone can think we have the address. 

Now, I'm entering a gibberish address so it might not happen to other users thinking they must have a legit address. 
However, is it possible to add a validation message that says "the address doesn't exist" type of thing? 
If it's possible, I will ask @kimchoi-jjiggae and add this to the PRD and create a ticket for it.

<img src="https://github.com/user-attachments/assets/933b3084-41ee-4f76-aa34-7bc7a7340483" width="320" />

Reply:
@shiozakiyuko - I answered in ticket - I prefer to keep such info in a single place, and the ticket seems like the place of record.